docs/spec/crisis/02_messages.md

```diff
@@ -0,0 +1,27 @@
+# Messages
+
+In this section we describe the processing of the crisis messages and the
+corresponding updates to the state. 
+
+## MsgVerifyInvariant
+
+Blockchain invariants can be check using the `MsgVerifyInvariant` message. 
+
+```golang
+type MsgCreateValidator struct {
+	Sender         sdk.AccAddress 
+	InvariantRoute string
+}
+```
+
+This message is expected to fail if: 
+ - the sender does not have enough coins for the constant fee
+ - the invariant route is not registered 
+
+This message checks the invariant provided, if the invariant is broken it
+panics halting the blockchain. If the invariant is broken, the constant fee is
+refunded to the message sender from the community pool, however if the
+invariant is not broken, the constant fee will not be refunded.
```

Comment:
```suggestion
invariant is not broken, the constant fee will not be refunded and will instead be burned.
```

Reply:
That's incorrect it is a fee given to the validators